src/commands/nft/usage.ts

```diff
@@ -0,0 +1,82 @@
+import Command from '../../common/base';
+import { cli } from 'cli-ux';
+import { formatBytes, formatNumbers } from '../../utils';
+import { NFTDailyStat, NFTUsage } from '../../types/nft';
+import NFTHttp from '../../rpc/nft';
+import { NodeDailyStat, NodeHourlyStat, NodeUsage } from '../../types';
+
+export const columns = {
+  date: {
+    header: 'Date',
+    minWidth: 13,
+    get: (row: any) => row.date,
+  },
+  usage: {
+    header: 'Usage',
+    minWidth: 13,
+    get: (row: any) => row.usage,
+  },
+};
+
+export default class Usage extends Command {
+  public static description = 'show nft api statistics';
+  public static examples = [`$ henesis nft:usage`];
+  public static flags = {};
+  public static args = [];
+
+  public async run(): Promise<void> {
+    try {
+      const [nftDailyStats] = await Promise.all([NFTHttp.dailyUsage()]);
+
+      if (nftDailyStats.length !== 0) {
```

Comment:
Wouldn't it be nice to show headlines and zero values only if there was no data?